.cicd/Jenkinsfile

```diff
@@ -122,17 +122,26 @@ pipeline {
                         }
                     }
 
-                    // Run the unified build script; if successful create a tarball of the build and upload to S3
-                    stage('Build') {
-                        steps {
-                            echo "Building SRW (${env.SRW_COMPILER}) on ${env.SRW_PLATFORM} (using ${env.WORKSPACE})"
-                            sh 'bash --login "${WORKSPACE}/.cicd/scripts/srw_build.sh"'
+                    parallel{
+                        // Run the unittest functional tests that require an HPC platform
+                        stage('Functional Tests) {
```

Comment:
Please add `'` to the end of `Functional Tests` on line 127:

`stage('Functional Tests') {`

This is causing the Jenkins test to fail during the initialization step.